MIGRATION-FROM-SCK.md

```diff
@@ -0,0 +1,325 @@
+### Checkpoint Translation
+WIth migration to the Splunk OpenTelemetry Collector for Kubernetes, the underlying framework/agent being used has changed, so there needs to be a baton handoff for checkpoint data so that the new OpenTelemetry agent can continue where Fluentd left off. All of this occurs automatically as an initContainer when you deploy the new helm chart in your cluster for the first time. As long as you properly configured the values.yaml file for the new helm chart, it will pick up where it left off. Without proper configuration for migration, there is a possibility of either data duplication or data loss while migrating from SCK to Splunk OpenTelemetry Collector for Kubernetes. 
+ 
+ If you need to migrate Fluentd's position files again, you can delete the OpenTelemetry checkpoint files in the ```"/var/lib/otel_pos/"``` directory from Kubernetes nodes. Then restart the new helm chart Daemonet.
+ 
+## Step 1: Preparing Your values.yaml file for migration
+You must translate the values.yaml file from SCK to an appropriate format for SCK-OTEL. Before we begin, here are the configurations for SCK and SCK-OTEL. 
+### Current Configuration for SCK
+<https://github.com/splunk/splunk-connect-for-kubernetes/blob/develop/helm-chart/splunk-connect-for-kubernetes/values.yaml>
+### Current Configuration for Splunk OpenTelemetry Collector for Kubernetes
+<https://github.com/andrewy-splunk/splunk-otel-collector-chart/blob/main/helm-charts/splunk-otel-collector/values.yaml>
+
+### Translating global/generic/splunk configurations from SCK to Splunk OpenTelemetry Collector for Kubernetes
+#### Specifying your Splunk Platform and HTTP Event Collector (HEC) configuration
+You can combine the "host", "port" and "protocol" options from SCK to use the "endpoint" option in Splunk OpenTelemetry Collector for Kubernetes. This option uses this format: "<http://X.X.X.X:8088/services/collector>" which will be interpreted as "protocol://host:port/services/collector".
+
+If you are using the "clientCert", "clientKey" and "caFile" options from SCK, you can use the corresponding "clientCert", "clientKey" and "caFile" options in Splunk OpenTelemetry Collector for Kubernetes to specify your HEC certificate chain.
+
+If you are using the "insecureSSL" option from SCK, you can use the insecure_skip_verify option in Splunk OpenTelemetry Collector for Kubernetes to specify whether to verify the certificates on HEC.
+
+If you are using the "indexName" option from SCK you can use the index option in Splunk OpenTelemetry Collector for Kubernetes to specify which index you want to index data into. Translating custom configs from SCK to Splunk OpenTelemetry Collector for Kubernetes for logs
```

Comment:
"If you are using the `indexName` option from SCK, use the index option in Splunk OpenTelemetry Collector for Kubernetes to specify which index you want to index data into."

The following sentence is incomplete:

"Translating custom configurations from SCK to Splunk OpenTelemetry Collector for Kubernetes for logs..."